src/plots/cartesian/layout_attributes.js

```diff
@@ -412,6 +412,16 @@ module.exports = {
             'To set ticks every 4 years, set `dtick` to *M48*'
         ].join(' ')
     },
+    ticklabeljump: {
+        valType: 'integer',
+        min: 0,
+        dflt: 0,
+        editType: 'ticks',
+        description: [
+            'Sets the step between ticklabels.',
+            'Could be used to hide labels between every n-th ticks.'
```

Comment:
You're right that `dticklabel` isn't a good name as it implies it should have the same units as `dtick` instead of being an integer multiplying `dtick`. I don't love `jump` though. Some thoughts:
- `ticksperlabel` (feels to me like the most precise name, and it's fairly concise, but it sort of makes it look like the ticks are modified rather than the labels)
- `skipticklabels` (but then it should mean "how many do we skip," ie 1 should mean "show every 2nd" and 0 should be the default behavior - add one to get the current meaning)
- any other suggestions?

If we keep the current meaning, the default and min should be 1 instead of 0. Either way the description could be made more explicit, something like:

> Sets the spacing between tick labels as compared to the spacing between ticks. A value of 1 (default) means each tick gets a label. A larger value n means only every nth tick is labeled. `tick0` determines which labels are shown.

Reply:
I uses `skipticklabels` in https://github.com/plotly/plotly.js/pull/6088/commits/2676d24ff511ddd69dfc25b9b4b0b7b3c23e62ba.
But how about `ticklabelpace`?

Reply:
Alright, let's stick with `skipticklabels` - I still don't love it but it's the best we have so far.